plugins/module_utils/vars.py

```diff
@@ -0,0 +1,260 @@
+# -*- coding: utf-8 -*-
```

Comment:
Why is the file named `vars.py`? It implements a `NewStructWith` class, it should be named in a way that is closer to what it does.

Reply:
The file that contain the `StructureMerging` class was named into `vars.py` to do like in this file <https://github.com/ansible/ansible/blob/devel/lib/ansible/utils/vars.py> in ansible core to create a module that group functions related to variable manipulation... Once again, may be this is not a "fresh" idea. I simply renamed this file into `structure_merging.py`.